packages/mui-joy/src/Select/Select.tsx

```diff
@@ -605,10 +609,28 @@ const Select = React.forwardRef(function Select<TValue extends {}>(
         {endDecorator && <SlotEndDecorator {...endDecoratorProps}>{endDecorator}</SlotEndDecorator>}
 
         {indicator && <SlotIndicator {...indicatorProps}>{indicator}</SlotIndicator>}
+        <input
+          name={name}
+          tabIndex={-1} // remove from tab order
+          aria-hidden="true" // hide from screen reader
+          required={required ? true : undefined}
+          value={getSerializedValue(selectedOption)}
+          onChange={noop} // to prevent React warning
+          style={{
```

Comment:
Nice! I'm wondering - since this is an element like any other, perhaps we could use `styled` to apply its styles, no?

Reply:
Or, and perhaps even better, if we use `style`, we could move this to `useSelect`'s `getInputProps` and have all implementations benefit from it.

Reply:
Agree! will update it.